fast64_internal/oot/c_writer/oot_level_c.py

```diff
@@ -444,13 +444,24 @@ def ootTransitionActorListToC(scene, headerIndex):
 	data.source += '};\n\n'
 	return data
 
+def ootRoomExternToC(room):
+	return "extern u8 " + room.roomName() + "SegmentRomStart[];\n" + \
+		"extern u8 " + room.roomName() + "SegmentRomEnd[];\n"
+
 def ootRoomListEntryToC(room):
 	return "{ (u32)_" + room.roomName() + "SegmentRomStart, (u32)_" + room.roomName() + "SegmentRomEnd },\n"
 
 def ootRoomListHeaderToC(scene):
 	data = CData()
-	data.header = "extern RomFile " + scene.roomListName() + "[];\n"
-	data.source = "RomFile " + scene.roomListName() + "[] = {\n"
+
+	# Write externs for rom segments
+	for i in range(len(scene.rooms)):
+		data.source += ootRoomExternToC(scene.rooms[i])
+	data.source += "\n"
```

Comment:
wouldn't this fit better in the header?

Reply:
Not really, it isn't needed by any other files

Reply:
I don't think the main point of the header is to have symbols for use by other files though,
for example the next line `data.header += "extern RomFile " + scene.roomListName() + "[];\n"` declares the room list in the header

Reply:
Well, eventually I planned on making another pr that merges the headers for each room and the scene into one file, which in my mind would make sense as something for external access rather than internal access. So since these symbols are intended to be used "internally" I thought this was more fitting.